_quarto.yml

```diff
@@ -0,0 +1,40 @@
+project:
+  type: book
+
+book:
+  title: "What They Forgot to Teach You About R"
+  subtitle: "The stuff you need to know about R, besides statistical analysis."
+  repo-url: https://github.com/rstats-wtf/what-they-forgot
+  site-url: https://rstats.wtf
+  author:
+    - Jennifer Bryan
+    - Jim Hester
+  date: "8/27/2022"
```

Comment:
Do we need a date field? I'm guessing no.

Reply:
I don't think so either.